### ISSUE TITLE
Refine Duplicate File Deletion Feature

### ISSUE THREAD

**Issue by ClickClack7531** [d4rken-org/sdmaid-se#911]
TLDR:
I really don't like how simplistic and unspecific the duplicate file remover feature is. I can't specify what file I would like to keep nor choose which ones I want deleted. It's also harder to view the duplicates that are detected and will be deleted.

The Rest:
What I would want is the ability to choose what files I want to be deleted, but also have the ability to automatically select files to be deleted under certain conditions like you could with the first SD Maid. I would like to view what files were detected individually and not in the form of a slideshow.

The first SD Maid pretty much has the best gui in my opinion, all the SD Maid FOSS needs is to improve how it interacts with the modern android operating system to index and layout these files. Sd Maid FOSS is very good at detecting duplicates, and very fast too. It's just the gui I don't like. Every time I press something, like the thumbnail, expecting it to expand the image for viewing, ot just asks "Would you like to delete all but 1 copy of this file" and it's annoying because sometimes I use the Perpetual Hash to detect images that are the same but different, and since I can't choose which one I want to keep, nor does it tell me which one it's keeping; I'm paranoid that I might be saving the lower quality copy of the image instead of the high quality one. Or the deeper nested one. Or the most recent/oldest one. Basic sorting that was in the first SD Maid is what I want.
